coverage/src/BamView-WebComponent.js

```diff
@@ -151,6 +151,19 @@ button:hover {
     fill: var(--data-color);
 }
 
+.loader-container {
```

Comment:
This should be `loading-indicator-container` to match the new name. Note that we now have 2 separate implementations of loading indicators. One in the coverage chart and the other in the percent and histograms. It would be nice if we could use a single loading indicator for all of them.

Reply:
That makes sense. I deleted the indicator on the main chart. Please see #72